### PR TITLE
YM-276 | Initial apollo federation gateway

### PR DESCRIPTION
**Pull request by charn**
This PR initializes the federation gateway for youth-membership project. Code is heavily influenced by https://github.com/City-of-Helsinki/berth-federation-gateway/

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "skipLibCheck": true
+    },
+    "exclude": [
+        "node_modules",
+        "dist"
+    ],
+    "include": [
+        "src/**/*"
+    ]
+}
```
Comment (igordavydsson) [inline tsconfig.json]: missing newline : D

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+.tern-port
+
+# Docker env
+docker-compose.env.yaml
+
+# Editors
+.idea
+.vscode/settings.json
+
+
```
Comment (igordavydsson) [inline .gitignore]: too many newlines : D

**Review by igordavydsson — APPROVED** [review]
Looks good! 👍 

thumbsup

File: .gitignore (hunk 2 above)
Comment (charn) [reply .gitignore]: Fixed all the newline issues. :sweat_smile:

**Pull request merged**